refiner-io/mobile-sdk-react-native
# Upgrade Native SDK versions and add setUser method

# What it does

1. This pull request upgrades Refiner iOS to 1.5.2 and Android SDK to 1.5.1
2. Adds setUser method

# How to test it

1. Implement setUser method
4. Run the app
5. Open app and see the survey shows up successfully

# Acceptance criteria

* No issues appears during project build
* Example app runs successfully
